eras/conway/impl/src/Cardano/Ledger/Conway/Governance.hs

```diff
@@ -1022,6 +1023,13 @@ finishDRepPulser (DRPulsing (DRepPulser {..})) =
         }
     !ratifyState' = runConwayRatify dpGlobals ratifyEnv ratifyState ratifySig
 
+-- | Force computation of DRep stake distribution and figure out the next enact
+-- state. This operation is useful in cases when access to new EnactState or DRep stake
+-- distribution is needed moer than once. It is safe to call this function at any
+-- point. Whenever pulser is already in computed state this will be a noop.
+forceDRepPulsingState :: ConwayEraGov era => NewEpochState era -> NewEpochState era
+forceDRepPulsingState nes = nes & newEpochStateDRepPulsingStateL %~ completeDRepPulsingState
+
```

Comment:
I think we should add a property test that checks that a pulser is always completed after applying `completeDRepPulsingState`

Reply:
I think it is obvious from implementation that this property will hold. Moreover this is a helper function for downstream users and ledger has no use for it, so I am not too worried about it